compiler/passes/initializerRules.cpp

```diff
@@ -2318,3 +2328,61 @@ FnSymbol* buildClassAllocator(FnSymbol* initMethod) {
 
   return fn;
 }
+
+static void buildTypeFunction(FnSymbol* fn) {
+  Symbol* _this = fn->_this;
+  AggregateType* at = toAggregateType(_this->type);
+
+  SET_LINENO(at);
```

Comment:
I ended up doing this later, I think the original line came about from looking at the creation of the _new function